src/style.css

```diff
@@ -0,0 +1,128 @@
+body {
+  margin: 0;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  line-height: 14px;
+}
+
+/* #region Stars */
```

Comment:
instead of providing #region better make separate files as per [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md)
> 10. [FILE STRUCTURE] - You should create separate file for each BEM block